go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -164,7 +164,9 @@ func init() {
 
 	flag.BoolVar(&enableReplicationReporter, "enable_replication_reporter", false, "Use polling to track replication lag.")
 	flag.BoolVar(&currentConfig.EnableOnlineDDL, "queryserver_enable_online_ddl", true, "Enable online DDL.")
-	flag.BoolVar(&currentConfig.SanitizeLogMessages, "sanitize_log_messages", false, "Remove potentially sensitive information in tablet INFO, WARNING, and ERROR log messages such as query parameters.")
+
+	flag.Int64Var(&currentConfig.RowStreamer.MaxTrxHistLen, "vreplication_copy_phase_max_trx_history", 1000000, "The maximum transaction history that can exist on a vstreamer (source) before starting another round of copying rows. This helps to limit the impact on the source tablet.")
```

Comment:
Should we call it `vreplication_copy_phase_max_history_length` or `vreplication_copy_phase_max_history_list_length` to mirror the use in mysql docs?

Reply:
They're getting pretty long, but I think I like:
 - vreplication_copy_phase_max_innodb_history_list_length
 - vreplication_copy_phase_max_mysql_replication_lag

What do you think?

Reply:
nice, long is fine: they are self-documenting!